### PR TITLE
fix(typo): package manager's C++ Archive Network (cppan) heading

### LINKED ISSUE #4117: [ Typo ]  Package Managers - C++ Archive Network (cppan) Heading
# Description:
I have come across a typo in the "Package Managers" section of the road map website. Specifically, under the heading for **C++ Archive Network (cppan)**, there is an issue with the formatting of the heading. I would like to rectify this issue by providing the necessary corrections in the Markdown format.

### PR DESCRIPTION
**Pull request by balaji-sivasakthi**
fix #4117

### CODE REVIEW

File: src/data/roadmaps/cpp/content/117-package-managers/index.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ For example, installing a package using vcpkg:
 ./vcpkg install boost:x64-windows
 ```
 
-##C++ Archive Network (cppan)
+## C++ Archive Network (cppan)
```
Comment (rtk-rnjn) [inline src/data/roadmaps/cpp/content/117-package-managers/index.md:31]: Good catch

**Comment by balaji-sivasakthi** [comment]
@kamranahmedse

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -1,5 +1,4 @@
 lockfileVersion: '6.0'
-
 settings:
   autoInstallPeers: true
   excludeLinksFromLockfile: false
```
Comment (arikchakma) [inline pnpm-lock.yaml:4]: Would you mind removing it from the PR?

Comment (balaji-sivasakthi) [reply pnpm-lock.yaml:4]: yes,  I will do it.

**Comment by balaji-sivasakthi** [comment]
@arikchakma updated!

**Comment by arikchakma** [comment]
> @arikchakma updated!

Thanks

**Pull request merged**